api/src/test/java/org/pmiops/workbench/workspaces/WorkspaceServiceTest.java

```diff
@@ -219,31 +219,35 @@ public void setUp() {
   }
 
   private RawlsWorkspaceResponse mockRawlsWorkspaceResponse(
-      String workspaceId,
-      String workspaceName,
+      String workspaceTerraUuid,
+      String workspaceTerraName,
       String workspaceNamespace,
       RawlsWorkspaceAccessLevel accessLevel) {
     RawlsWorkspaceDetails mockWorkspace =
         new RawlsWorkspaceDetails()
-            .workspaceId(workspaceId)
-            .name(workspaceName)
+            .workspaceId(workspaceTerraUuid)
+            .name(workspaceTerraName)
             .namespace(workspaceNamespace);
 
     RawlsWorkspaceResponse mockWorkspaceResponse = new RawlsWorkspaceResponse();
     mockWorkspaceResponse.workspace(mockWorkspace);
     mockWorkspaceResponse.accessLevel(accessLevel);
+
+    doReturn(mockWorkspaceResponse)
```

Comment:
Was this not needed before?

Reply:
There were two `mockRawlsWorkspaceResponse()` methods previously, and this is the result of merging them.  The one github chose as the diff didn't have it, but the other one did.

Considering the name of the method, it's strange that one of them didn't actually mock the rawls workspace response!